promotion/suggestions.go

```diff
@@ -207,6 +210,16 @@ func (service *Service) Suggest(ctx context.Context, credentials []CredentialBin
 		return err
 	}
 
+	for _, funding := range fundingTypes {
+		value, _ := total.Float64()
+		labels := prometheus.Labels{
+			"type":    suggestion.Type,
+			"funding": funding,
+		}
+		countContributionsTotal.With(labels).Inc()
```

Comment:
hmm this is kind of tricky - should one contribution using multiple funding sources really be counted once for each funding source? if we sum over the different funding sources then the total won't match.

maybe we actually need three different contribution quantities?
```
contributions_bat_total - broken down by type and funding source (no change other than the incorrect total mentioned above)
contributions_total - total number of contributions broken down by type
contributions_including_funding_source_total - total number of contributions broken down by type and funding source (as done here, with "double" counts)
```